metpy/plots/skewt.py

```diff
@@ -522,6 +523,107 @@ def plot_mixing_lines(self, w=None, p=None, **kwargs):
         kwargs.setdefault('alpha', 0.8)
         return self.ax.add_collection(LineCollection(linedata, **kwargs))
 
+    def shade_area(self, y, x1, x2=0, which='both', **kwargs):
+        r"""Shades areas of CAPE.
+
+        Shades areas where the parcel is warmer than the environment (areas of positive
```

Comment:
Need to update the docstring--I don't think you ever went back after copy/pasting:
- Not just CAPE.
- No documentation of `which`, mention p,T,T_parcel which aren't actually parameters.